README.md

```diff
@@ -38,91 +36,147 @@
-  without node features, using the Node2Vec [1] and Metapath2Vec [2] algorithms.
-  - See the demo in folder `demos/link-prediction-graphsage` for an example of how to predict the existence of links between
-  nodes with node features using the GraphSAGE [3] algorithm.
+If you get stuck or have a problem, there's many ways to make progress and get help or support:
+
+- [Read the documentation](https://stellargraph.readthedocs.io)
+- [Consult the examples](https://github.com/stellargraph/stellargraph/tree/master/demos/)
+- Contact us:
+  - [Ask questions and discuss problems on the StellarGraph Discourse forum](https://community.stellargraph.io)
+  - [File an issue](https://github.com/stellargraph/stellargraph/issues/new/choose)
+
+
+## Example: GCN
+
+One of the earliest deep machine learning algorithms for graphs is a Graph Convolution Network (GCN) [6]. The following example uses it for node classification: predicting the class from which a node comes. It shows how easy it is to apply using StellarGraph, and shows how StellarGraph integrates smoothly with Pandas and TensorFlow and libraries built on them.
+
+#### Data preparation
+
+Data for StellarGraph can be prepared using common libraries like Pandas and scikit-learn.
+
+``` python
+import pandas as pd
+from sklearn import model_selection
+
+# load data into Pandas DataFrames, e.g. from CSV files or a database
+nodes, edges, targets = load_my_data()
+
+# Use scikit-learn to compute training and test sets
+train_targets, test_targets = model_selection.train_test_split(targets, train_size=0.5)
+
+```
+#### Graph machine learning model
 
-* Recommender systems
-  - See the demo in folder `demos/link-prediction-hinsage` for an example of how to predict
-  movie ratings between users and movies using a Heterogeneous generalisation of GraphSAGE model, which we call HinSAGE.
+This is the only part that is specific to StellarGraph. The machine learning model consists of some graph convolution layers followed by a layer to compute the actual predictions as a TensorFlow tensor. StellarGraph makes it easy to construct all of these layers via the `GCN` model class. It also makes it easy to get input data in the right format via the `StellarGraph` graph data type and a data generator.
 
--->
+```python
+import stellargraph as sg
 
+# convert the raw data into StellarGraph's graph format for faster operations
+graph = sg.StellarGraph(nodes, edges)
+
+generator = sg.mapper.FullBatchNodeGenerator(graph, method="gcn")
+
+# two layers of GCN, each with hidden dimension 16
+gcn = sg.layer.GCN(layer_sizes=[16, 16], generator=generator)
+x_inp, x_out = gcn.build() # create the input and output TensorFlow tensors
+
+# use TensorFlow Keras to add a layer to compute the (one-hot) predictions
+predictions = tf.keras.layers.Dense(units=len(ground_truth_targets.columns), activation="softmax")(x_out)
```

Comment:
The `import tensorflow as tf` comes later. This line should come after that.

A block that states all the steps about building the model should be together. 
Also,  I think model building part with the prediction layer should come together. 
Since `model = tf.keras.Model(inputs=x_inp, outputs=predictions)` needs `x_input` and `predictions` so it would be less confusing if they are in one place together.

Reply:
In the attached screenshot, the model definition comes as a first line of the 2nd block. To me it would look more appropriate it this was not cut-off from the 3 last lines of the previous block. 
Like, the model is taking as input from the layers defined above. So they all seem to be a complete thing together. 

<img width="791" alt="Screen Shot 2020-03-23 at 5 54 17 pm" src="https://user-images.githubusercontent.com/26869354/77290173-b6717200-6d2f-11ea-884b-8cfc9770844d.png">

Reply:
Sounds good, I'll move the `model = ` line and tweak the text a bit.